docs/sqlserver.md

```diff
@@ -0,0 +1,55 @@
+# `sqlserver` Metrics Receiver
+
+The `sqlserver` metrics receiver can fetch stats from a SQL Server via the [Windows Performance Counters](https://docs.microsoft.com/en-us/windows/win32/perfctrs/performance-counters-portal).
+
+## Prerequisites
+Because this `sqlserver` receiver makes use of the windows perforamnce counters, it is required for it to run locally on the same machine with the SQL Server installation.
+
+## Configuration
+
+| Field                 | Default            | Description |
+| ---                   | ---                | ---         |
+| `type`                | required           | Must be `sqlserver`. |
+| `collection_interval` | `60s`              | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+
+Example Configuration:
+
+```yaml
+metrics:
+  receivers:
+    sqlserver:
+      type: sqlserver
+      collection_interval: 60s
+  service:
+    pipelines:
+      sqlserver:
+        receivers:
+          - sqlserver
+```
+
+## Metrics
+The Ops Agent collects the following metrics from SQL Server.
+Note: In order for any metrics to be retrieved, the receiver must run locally on the same server as the SQL Server installation.
+
+| Metric                                               | Data Type      | Unit        | Labels      | Description |
+| ---                                                  | ---            | ---         | ---         | ---         | 
+| workload.googleapis.com/sqlserver.user.connection.count | Gauge       | connections |             | Number of users connected to the SQL Server. |
+| workload.googleapis.com/sqlserver.lock.wait_time.avg | Gauge          | ms          |             | Average wait time for all lock requests that had to wait. |
+| workload.googleapis.com/sqlserver.lock.wait.rate     | Gauge          | requests/s  |             | Number of lock requests resulting in a wait. |
```

Comment:
The descriptions of these metrics say `Number of …`, but they are actually rates. Do we need to mention something about that in the description?

Alternatively, should (can?) we ingest these as sums and let the users compute the rates at query time?

Reply:
@igorpeshansky Unfortunately all of the sql server windows performance counters are gauges. The descriptions were mostly pulled verbatim from Microsoft's documentation for the counters (even though again they are gauges), but we could change the descriptions easily enough if need be.

Reply:
IMO the types and units make it obvious enough that these are rates. I don't view the descriptions as contradictory but I'm not against changing it to something like `Number of lock requests per second resulting in a wait.` if we want it to be more clear or if it's more consistent with how we document other such receivers.